index.md

```diff
@@ -1,12 +1,15 @@
-## See installation instructions for:
+## JupyterHub helm charts
+{:.no_toc}
+
+See installation instructions for:
 
 - [JupyterHub](https://zero-to-jupyterhub.readthedocs.io)
 - [BinderHub](https://binderhub.readthedocs.io)
 
 Jump to:
-{%- for chartmap in site.data.index.entries %}
-- [Development Releases: {{ chartmap[0] }}](#development-releases-{{ chartmap[0] | slugify }})
-{%- endfor %}
+
+- this list will be replaced with the table of contents
+{:toc}
```

Comment:
I don't understand these changes, but if it works go for it!

Reply:
This is [part of kramdown](https://kramdown.gettalong.org/converter/html.html#toc), which is the markdown implementation used by jekyll/pages